src/styles/blocks/card.scss

```diff
@@ -0,0 +1,106 @@
+.card {
+  width: 200px;
+  border-radius: 5px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  flex-direction: column;
+  display: flex;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    padding-left: 19px;
+    padding-top: 32px;
+  }
+
+  &__info {
+    margin: 40px 16px 0;
+
+    &-name {
+      font-weight: 500;
+      font-size: 12px;
+      line-height: 18px;
+      color: #060b35;
+      font-style: normal;
+    }
+
+    &-code {
+      font-style: normal;
+      font-weight: normal;
+      font-size: 10px;
+      line-height: 14px;
+      color: #616070;
+      margin-top: 4px;
+      left: 16px;
+    }
+
+    &-review-wrapper {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      width: 166px;
+      height: 16px;
+      margin-top: 16px;
+
+    }
+
+    &-review-count {
+      font-size: 10px;
+      line-height: 14px;
+      font-style: normal;
+      font-weight: normal;
+      display: flex;
+      align-items: flex-end;
+    }
+
+    &-price {
+      display: flex;
+      justify-content: space-between;
+      align-items: flex-end;
+      width: 166px;
+      height: 18px;
+      margin-top: 24px;
+
+      &-letter {
+        font-size: 12px;
+        line-height: 18px;
+        display: flex;
+        align-items: flex-end;
+        color: rgba(97, 96, 112, 1);
+      }
+
+      &-number {
+        font-size: 16px;
+        line-height: 18px;
+        font-style: normal;
+        font-weight: 700;
+        display: flex;
+        align-items: flex-end;
+        text-align: right;
+      }
+    }
+  }
+  &__buy {
+    box-sizing: border-box;
```

Comment:
box-sizing should be the same for all elements, use * selector for that
```suggestion
```

Reply:
fix all, but cant use box-sizing: border-box for, they used only when its need